lib/event_icon_builder.rb

```diff
@@ -0,0 +1,36 @@
+class EventIconBuilder
```

Comment:
Yeah, I know I'm in the minority on this, but I always end up thinking it's extra dressing where all you need is a json blob in a controller. Most people out it in the view and use syntax helpers, which I'm not opposed to, but this has always felt simpler. And when rails5 API mode took away the views folder I felt emboldened